src/theme/components/header.ts

```diff
@@ -1,37 +1,17 @@
-const notification = {
-  bg: "#fee249",
-  width: "100%",
-  "div > .css-1jloxoz": {
-    fontSize: "var(--nypl-fontSizes-text-tag)",
-    letterSpacing: ".01em",
-    lineHeight: "1.25em",
-    margin: "0.5em 1.2em 0.5em 1.2em",
-  },
-  a: {
-    _focus: {
-      outlineColor: "#135772",
-      outlineStyle: "solid",
-      outlineWidth: "0.1875em",
-    },
-  },
-};
-
 const horizontalRule = {
   bg: "#ed1c24",
-  position: "relative",
-  top: -4,
+  marginTop: "0",
 };
 
 const Header = {
-  parts: ["container", "notification", "logo", "horizontalRule"],
+  parts: ["container", "horizontalRule", "logo"],
   baseStyle: ({ isWidthMobile, isWidthLarge }) => ({
```

Comment:
Let me know if there's a better place to ask this - why use `isWidthMobile` and `isWidthLarge` rather than [Chakra's responsive array syntax?  ](https://chakra-ui.com/docs/styled-system/features/responsive-styles)

Reply:
Yeah, it should. I guess building this rather quickly overlooked that but I can update.

Reply:
I've got the same question about https://github.com/NYPL/nypl-design-system/pull/1047 - perhaps this refactor can happen on top of this as well?

Reply:
Yes. Do you mean same PR or a subsequent one?